src/common/sha2/sha2_c.c

```diff
@@ -542,8 +548,10 @@ void oqs_sha2_sha384_inc_init_c(sha384ctx *state) {
 }
 
 void oqs_sha2_sha512_inc_init_c(sha512ctx *state) {
-	state->ctx = OQS_MEM_checked_malloc(PQC_SHA512CTX_BYTES);
-
+	state->ctx = OQS_MEM_malloc(PQC_SHA512CTX_BYTES);
```

Comment:
Why this change? It seems to just silence a (imo somewhat helpful) warning output.